### PR TITLE
hyperopt 0.2.7

### PR DESCRIPTION
**Pull request by skupr-anaconda**
License: https://github.com/hyperopt/hyperopt/blob/0.2.7/LICENSE.txt
Changelog: https://github.com/hyperopt/hyperopt/tags
Requiements:
- https://github.com/hyperopt/hyperopt/blob/0.2.7/setup.py
- http://hyperopt.github.io/hyperopt/setup/installation-notes/

Actions:
1. Add abs.yaml (disabled channels before merging)
2. Add --no-deps
3. Add numpy, setuptools, and wheel to host
4. Add `run_constrained`: pymongo, pyspark, lightgbm, scikit-learn
5. Add hyperopt.pyll to test/imports

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,35 +10,44 @@ source:
...
   entry_points:
     - hyperopt-mongo-worker=hyperopt.mongoexp:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
 
 requirements:
   host:
+    - python
+    - numpy
```
Comment (katietz) [inline recipe/meta.yaml]: Please add here the '={{ numpy }}' constrain to it

**Review by katietz — CHANGES_REQUESTED** [review]
beside this pinning thing for numpy, looks good to me

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Done

**Review by katietz — APPROVED** [review]
Thank you!

**Review by varlackc — COMMENTED** [review]
Just curious if we should remove the `abs.yaml` file in this context.

https://github.com/AnacondaRecipes/hyperopt-feedstock/blob/aa75e9aecdde71afe42f5f06f4ecabd06b3181c2/abs.yaml

**Comment by skupr-anaconda** [comment]
> Just curious if we should remove the `abs.yaml` file in this context.
> 
> https://github.com/AnacondaRecipes/hyperopt-feedstock/blob/aa75e9aecdde71afe42f5f06f4ecabd06b3181c2/abs.yaml

`aggregate_check: false` will be removed other keys will be enabled for uploading artifacts

**Comment by akabanovs** [comment]
LGTM

**Review by akabanovs — APPROVED** [review]
LGTM

**Pull request merged**